src/shared/components/Banners/RebrandBanner/RebrandBanner.tsx

```diff
@@ -0,0 +1,141 @@
+    ? `rebrand-banner-dismissed-${userId}:${workspaceId}`
+    : `rebrand-banner-dismissed-${userId}`;
+
+// Global key for anonymous users (e.g., on the login screen)
+export const GLOBAL_DISMISSED_KEY = 'rebrand-banner-dismissed-global';
+
+const DISPLAY_ROUTES = [
+  '/auth',
+  '/dashboard/applets',
+  '/dashboard/managers',
+  '/dashboard/respondents',
+];
+
+export const RebrandBanner = (props: BannerProps) => {
+  const { ownerId } = workspaces.useData() ?? {};
+  const userData = auth.useData();
+  const userId = userData?.user.id;
+  const location = useLocation();
+
+  const [isRebrandBannerActive, setIsRebrandBannerActive] = useState(false);
+  const [isCollapsing, setIsCollapsing] = useState(false);
+
+  const handleOnExited = () => {
+    if (isCollapsing) {
+      setIsCollapsing(false);
+      setIsRebrandBannerActive(false);
+    }
+  };
+
+  useEffect(() => {
+    // For auth screen or when user is not logged in yet
+    if (!userId) {
+      const globalDismissed = localStorage.getItem(GLOBAL_DISMISSED_KEY);
+      setIsRebrandBannerActive(!globalDismissed);
+
+      return;
+    }
+
+    // For logged-in users
+    const userDismissed = localStorage.getItem(getDismissedKey(userId));
+
+    // If there's a workspace, also check workspace-specific dismissal
+    if (ownerId) {
+      const workspaceDismissed = localStorage.getItem(getDismissedKey(userId, ownerId));
+      setIsRebrandBannerActive(!userDismissed && !workspaceDismissed);
+    } else {
+      setIsRebrandBannerActive(!userDismissed);
+    }
+  }, [userId, ownerId]);
+
+  const isDisplayRoute = DISPLAY_ROUTES.some(
+    (route) =>
+      location.pathname === route ||
+      (location.pathname.startsWith(route) && route.includes('/auth')),
+  );
+
+  const handleDismiss = () => {
+    // For auth screen or when user is not logged in yet
+    if (!userId) {
+      localStorage.setItem(GLOBAL_DISMISSED_KEY, 'true');
+    } else {
+      // Set user-level dismissal
+      localStorage.setItem(getDismissedKey(userId), 'true');
+
+      // Also set workspace-specific dismissal if available
+      if (ownerId) {
+        localStorage.setItem(getDismissedKey(userId, ownerId), 'true');
+      }
+    }
+
+    setIsCollapsing(true);
+  };
+
+  // Don't render if not on a display route or if no user info is available on non-auth routes
+  const shouldRender = isDisplayRoute && (location.pathname.startsWith('/auth') || userId);
+
+  return shouldRender ? (
+    <Box>
+      <Collapse in={isRebrandBannerActive && !isCollapsing} enter={false} onExited={handleOnExited}>
+        {isRebrandBannerActive && (
+          <Banner
+            duration={null}
+            severity={undefined}
+            data-testid="rebrand-banner"
+            onClose={(reason) => {
+              if (reason === 'manual') {
+                handleDismiss();
+              }
+            }}
+            icon={<StyledImg src={curiousIcon} />}
+            sx={{
+              backgroundColor: '#000',
```

Comment:
Nit: Super minor shade difference in Figma…
```suggestion
              backgroundColor: '#0B0907',
```

Reply:
Nice spot! Hadn't noticed looking at it in my monitor. Got a get a more color-accurate one 😅